examples/testing/swiss-knife/src/main/scala/swissknife/FlinkCounter.scala

```diff
@@ -0,0 +1,27 @@
+package swissknife
+
+import cloudflow.flink.FlinkStreamlet
+import org.apache.flink.streaming.api.scala._
+import cloudflow.streamlets.StreamletShape
+import cloudflow.streamlets.avro._
+import cloudflow.flink._
+
+class FlinkCounter extends FlinkStreamlet {
+
+  @transient val in  = AvroInlet[Data]("in")
+  @transient val out = AvroOutlet[Data]("out", _.src)
+
+  @transient val shape = StreamletShape.withInlets(in).withOutlets(out)
+
+  override def createLogic() = new FlinkStreamletLogic {
+    override def buildExecutionGraph = {
+      val stream: DataStream[Data] =
```

Comment:
I remember that @skyluc had a very simple flink streamlet, that did not catch the issue that we saw, "because it was too simple". This might be the case here? (I'm not sure but might be interesting to chat about it)

Reply:
@skyluc what was the issue that Ray refers to?